packages/compass-crud/src/components/table-view/breadcrumb.jsx

```diff
@@ -37,7 +37,11 @@ class BreadcrumbComponent extends React.PureComponent {
           onClick={this.onHomeClicked.bind(this)}
           className={`${BEM_BASE}-tab`}
         >
-          <FontAwesome name="home" className={`${BEM_BASE}-home-icon`} />
+          <Icon
+            glyph="Home"
+            size={11}
```

Comment:
[Created a follow up to move this to xsmall](https://jira.mongodb.org/browse/COMPASS-6306)